source/adapters/cuda/event.cpp

```diff
@@ -80,6 +82,8 @@ bool ur_event_handle_t_::isCompleted() const noexcept {
     }
   }
   return true;
+} catch (...) {
+  return exceptionToResult(std::current_exception()) == UR_RESULT_SUCCESS;
```

Comment:
rather than catching we could just remove the UR_CHECK_ERROR and leave it to hit `return false`, we aren't getting any more information out of the function by comparing against the UR error code as well

Reply:
I was about to do this change, but it seems that the `UR_CHECK_ERROR` macro prints out the error message:
https://github.com/oneapi-src/unified-runtime/blob/main/source/adapters/cuda/common.cpp#L38.
Should I remove it regardless?

Reply:
ah, we should probably leave it in then